infra/gravity/node_commands.go

```diff
@@ -568,11 +568,10 @@ const (
 // runOp launches specific command and waits for operation to complete, ignoring transient errors
 func (g *gravity) runOp(ctx context.Context, command string, env map[string]string) error {
 	var code string
-	executablePath := filepath.Join(g.installDir, "gravity")
+	sudoGravity := fmt.Sprintf(`cd %v && sudo -E ./gravity`, g.installDir)
```

Comment:
I find this `cd {{ .installDir }} && sudo ./gravity` pattern a bit odd, but it is consistent with what is in most of the rest of node_commands.go.

Some totally ad-hoc stats:

```
walt@work:~/git/robotest$ git checkout master
Switched to branch 'master'
Your branch is up to date with 'grav/master'.
walt@work:~/git/robotest$ grep -c 'filepath.Join(g.installDir, "gravity")' infra/gravity/node_commands.go
2
walt@work:~/git/robotest$ grep -c 'cd .* && .* \./gravity' infra/gravity/node_commands.go                
6
```

I considered unifying these to use a common code path, as there tends to be a bit of drift between different invocations (e.g. a missing `sudo` here, `-E` in some places) that is likely cause for future inconsistencies/bugs along the lines of #194.  However that greatly increased the scope of this PR.  I'd be glad to revisit if y'all think it is worthwhile.